pkg/service/job_service.go

```diff
@@ -17,7 +17,7 @@ func (s *Service) GetJobs() ([]model.Job, error) {
 }
 
 func (s *Service) CreateJob(job *model.Job) (*model.Job, error) {
-	result := s.DB.Create(&job)
+	result := s.DB.FirstOrCreate(&job, model.Job{JobId: job.JobId})
 
 	if err := result.Error; err != nil {
```

Comment:
`if result.Error != nil` ?

Reply:
No, i mean that you can simplify the check. No need to assign it to err just to check it.